website/docs/reference/widgets/select.md

```diff
@@ -127,15 +127,44 @@ Style properties allow you to change the look and feel of the widget.
 | -------------------- | -------------------------------------------------------- |
 | **Label Text Color** | Allows you to set text color for the label.              |
 | **Label Text Size**  | Allows you to set the size of the label.                 |
-| **Label Font Style** | Allows you to choose a font style, i.e., bold or italic. |
+| **Label Font Style** | Allows you to choose a font style, such as bold or italic. |
 | **Border Radius**    | Allows you to define curved corners.                     |
 | **Box Shadow**       | Allows you to choose from the available shadow styles.   |
 
-### Server-side Filtering
+### Server-side filtering
 
-The Select widget has the option to configure server-side filtering, where search queries are sent to the backend, and responses are used to populate options on the Select widget. When enabling server-side filtering in the widget, please update the default value to contain both `label` and `value` in this format `{"label":<label>, "value": <value>}` if the default value is not present in the default options.
+The Select widget has the option to configure server-side filtering, where search queries are sent to the backend, and responses are used to populate options on the Select widget. When enabling server-side filtering in the widget, please update the default value to contain both `label` and `value` in this format `{"label":<label>, "value": <value>}` if the default value isn't present in the default options.
 
 The video below explains how to configure this.
 
 <VideoEmbed host="youtube" videoId="QDmTwRaLzHg" title="Server-side Filtering" caption="Server-side Filtering"/>
 
+As shown in the video above, follow these steps to configure server side filtering:
```

Comment:
I think we could remove `above`, and the meaning would still be the same.

Reply:
Instead of `above` we can add the video's caption if that helps.

Reply:
Github shows for me that "above" has already been removed, though I can also add a reference to the video's caption for clarity